portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/rollingrestart/RollingRestartEE.testcase

```diff
@@ -886,15 +886,6 @@ definition {
 
 		User.logoutPG(specificURL = "http://localhost:9080");
```

Comment:
Before we only needed to logout because we have to login again after, but since our next step is shutting down the app server, this step is un-needed now.